workshop.md

```diff
@@ -91,162 +95,173 @@ Once you are in the new Lakehouse, create a new notebook by selecting `Open Note
+> **NB:** Please Note you can use any other Microsoft Learn module URL, but while copying the URL, make sure you are inside the `include` folder for this to work. i.e.
+> - `https://github.com/MicrosoftDocs/learn/tree/main/learn-pr/wwl-data-ai/analyze-images-computer-vision/includes`
+> - `https://github.com/MicrosoftDocs/learn/tree/main/learn-pr/wwl-data-ai/automl-azure-databricks/includes`
 
-![Screenshot of the Microsoft Learn GitHub Get started with Real-Time Analytics in Microsoft Fabric learn module](assets/microsoft-learn-github-kusto.png)
 
-To obtain the file URL, select the `index.yml` on GitHub and select the `Raw` button. Copy the URL since you'll use it in the code below.
+#### Install Dependencies
 
-Add the following code to your notebook into a new cell:
+add the following code to the first cell to install qrcode library
 
 ```python
-import requests
-import os
-import yaml
-
-# learn module metadata URL
-LEARN_GITHUB_BASE = "https://raw.githubusercontent.com/MicrosoftDocs/learn/main/learn-pr"
-url = f"{LEARN_GITHUB_BASE}/wwl/get-started-kusto-fabric/index.yml"
-
-# load module metadata
-response = requests.get(url)
-index_data = yaml.safe_load(response.content)
-
-# print the module metadata
-index_data
+%pip install qrcode
 ```
+We are going to use the qrcode library to generate the QR codes for the flashcards. We do this as a first step, because once the library is installed, the notebook will automatically restart the kernel to make the library available for the rest of the code.
 
-Run the cell.
-
-<div class="warning" data-title="Note">
-
-> Going forward, it is expected that you run each cell in the notebook to execute the code.
-
-</div>
-
-Now that the YAML file is loaded in the `index_data` variable, we can use it to build the folder structure in our Lakehouse.
-
+### Copy the data to your Lakehouse
+Add the following code to your notebook into a new cell:
 ```python
-# use the module uid to calculate the folder structure of the learn module
-module_uid = index_data['uid']
-
-module_units = index_data['units']
-
-# convert the module uid to a path
-uid_to_path = module_uid.replace(".", "/")
-
-# Path to Lakehouse Files
-LAKEHOUSE_FILE_PATH="/lakehouse/default/Files"
-
-# Path to Markdown Files
-MARKDOWN_PATH=f"{LAKEHOUSE_FILE_PATH}/markdown"
-
-# Path to Learn Module
-LEARN_MODULE_PATH=f"{MARKDOWN_PATH}/{uid_to_path}"
+# 1. Set up variables and paths
+from notebookutils import mssparkutils
+import requests
+import os
+import re
+import qrcode
 
-# create folder to store our Markdown files
-os.makedirs(LEARN_MODULE_PATH, exist_ok=True)
+# ----------------- User Input -----------------
+
+# User Input: GitHub URL (Please make sure the Ms Learn URL Repo have includes folder ie below URL)
```

Comment:
It's either MS Learn or Microsoft Learn. I'm inclined to the full spelling

Reply:
Renamed MS Learn to Microsoft Learn